src/main/scala/faces/apps/ModelViewer.scala

```diff
@@ -355,7 +355,7 @@ case class SimpleModelViewer(
     if (fc.showSaveDialog(null) == JFileChooser.APPROVE_OPTION) {
       var file = fc.getSelectedFile
       if (file.isDirectory) file = new File(file,"instance.ply")
-      if ( !file.getName.endsWith(".ply")) file = new File( file+".ply")
+      if ( !file.getName.endsWith(".ply")) file = new File( file.getAbsolutePath()+".ply")
```

Comment:
Could you please remove the empty parenthesis? This will fix a warning in the IDE (intellij).